src/UmaCtfAdapter.sol

```diff
@@ -347,6 +345,12 @@ contract UmaCtfAdapter is IUmaCtfAdapter, Auth, BulletinBoard, IOptimisticReques
             yesOrNoIdentifier, questionData.requestTimestamp, questionData.ancillaryData
         );
 
+        // If the OO returns the ignore price, reset the question
+        if (price == _ignorePrice()) {
+            questionData.reset = false;
```

Comment:
What if we just make reset non-conditionally reset the question, and put the logic to check `reset == false` in `priceDisputed` ?